packages/core/src/RenderingEngine/VolumeViewport.ts

```diff
@@ -89,6 +89,9 @@ class VolumeViewport extends Viewport implements IVolumeViewport {
   }
 
   private initializeVolumeNewImageEventDispatcher(): void {
+    const volumeNewImageHandlerBound = volumeNewImageHandler.bind(this);
```

Comment:
If you just replace all the following methods to arrow functions, you won't have to use any of these bindings and can just use the function definitions directly:

- `initializeVolumeNewImageEventDispatcher`
- `volumeNewImageHandler`
- `volumeNewImageCleanUp`